.codespell.skip

```diff
@@ -0,0 +1 @@
+./manifests/charts/ai-gateway-helm/crds/*
```

Comment:
if this is needed, then the spell check seems not running properly on the code comments in api.go? Since the typo `maxmimum` in api.go is the reason why https://github.com/envoyproxy/ai-gateway/actions/runs/12387688325/job/34577531455 is failing i guess?

Reply:
This comes from upstream Gateway HTTPRoute.

Reply:
aha ok, now that makes sense... thank you for clarifying huabing!